trillian.proto

```diff
@@ -217,8 +217,9 @@ message SignedLogRoot {
   int64 tree_size = 3;
   sigpb.DigitallySigned signature = 4;
 
-  int64 log_id = 5;
+  reserved 5; // (moved to key_hint)
   int64 tree_revision = 6;
+  bytes key_hint = 7;
```

Comment:
The `trillian.proto` file is effectively the external API definition, so it would be good to have a comment explaining what's in the `bytes`, and what format it's in.

Reply:
Good idea.